inc/assets.php

```diff
@@ -31,9 +31,9 @@ function bootstrap() : void {
  */
 function enqueue_vega() : void {
```

Comment:
Should we change this function name to `register_vega()` since we're changing what's doing?

Reply:
Good catch, I've made that change @pamprn09